object LetError1
  fn test(): Int = {
    val x: Int = 1;
    false
  }
end LetError1
